src/igvfd/schemas/changelogs/analysis_set.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`analysis_set.json`*
 
+### Schema version 7
+
+* Rename `file_set_type` `primary analysis` to `principal analysis`.
```

Comment:
The changelog should follow the pattern "Adjust [property name] enum list to replace [old enum] with [new enum]."